src/setupProxy.js

```diff
@@ -8,6 +8,11 @@ module.exports = (app) => {
       changeOrigin: true,
       secure: false,
       ws: false,
+      onProxyReq: (proxyReq, req, res) => {
+        if (req.headers['x-jm-authorization']) {
+          proxyReq.setHeader('authorization', req.headers['x-jm-authorization'])
+        }
+      },
```

Comment:
Should we add this to the web socket proxy as well right away?

Reply:
I don't know to be honest. Has never worked for me and is therefore not tested at all. I refrained from adding it, as I have no way to test it. Tend to think it is out of scope. Is this okay for you?